doc/security/security-overview.rst

```diff
@@ -551,27 +552,27 @@ following is the current proposal:
 -  Security sensitive bugs are only accessible (view/modify) to members
    of the Security Group; members of this Security Group are:
 
-   -  members of the Security Subcommittee
+   -  members of the Zephyr Security Subcommittee
```

Comment:
This list should just describe who the members are, so rewrite as:
```
   -  members of the Zephyr Security Subcommittee
   -  others, as proposed and ratified by the Zephyr Security Subcommittee
   -  the reporter

-  Members of this Security Group have the authority to add or remove other
   users for individual issues.   
```
Note that last bullet is outdented to belong to the outer bullet list (not part of the list
defining who the members are).